src/plugins/shared_ux/public/components/page_template/no_data_page/no_data_card/no_data_card.stories.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { NoDataCard } from './no_data_card';
+import { NoDataPageActions } from './types';
+
+export default {
+  title: 'No Data Card',
+  description: 'A wrapper around EuiCard, to be used on NoData page',
+};
+
+type Params = Pick<NoDataPageActions, 'recommended' | 'button' | 'description'>;
+
+export const PureComponent = (params: Params) => {
+  return (
+    <div style={{ width: '50%' }}>
+      <NoDataCard title={'You have no data'} {...params} />
+    </div>
+  );
+};
+
+PureComponent.argTypes = {
+  recommended: {
+    control: 'boolean',
+    defaultValue: true,
```

Comment:
I would keep this defaulted to `false` because this is actually a leftover from an old design that was decided against (for now).

```suggestion
    defaultValue: false,
```